packages/react/src/analytics/integrations/Riskified/Riskified.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  EventData,
+  integrations,
+  LoadIntegrationEventData,
+  StrippedDownAnalytics,
+  TrackTypes,
+  TrackTypesValues,
+  utils,
+} from '@farfetch/blackout-analytics';
+import get from 'lodash/get';
+import type { RiskifiedIntegrationOptions } from './types';
+
+class Riskified extends integrations.Integration<RiskifiedIntegrationOptions> {
+  baseUrl!: string;
+  isReadyPromise: Promise<unknown>;
+  resolveIsReadyPromise?: (value?: unknown) => void;
```

Comment:
Since `resolveIsReadyPromise` will be set in the constructor, you can change that to this:

```suggestion
  resolveIsReadyPromise!: (value?: unknown) => void;
```

And remove line 32.